src/models/Follower.model.ts

```diff
@@ -0,0 +1,41 @@
+import { ObjectId } from "mongodb";
+import mongoose, { Schema, Document } from "mongoose";
+import paginate from "mongoose-paginate-v2";
+
+export interface Follower extends Document {
+  followerId: ObjectId;
+  ownerId: ObjectId;
+  createdAt: string;
```

Comment:
```suggestion
  createdA?t: string;
```
We did not ask that, we filled it by default